editor/src/components/canvas/dom-walker.ts

```diff
@@ -324,16 +366,17 @@ export function useDomWalker(props: DomWalkerProps): React.Ref<HTMLDivElement> {
       // This assumes that the canvas root is rendering a Storyboard fragment.
       // The necessary validPaths and the root fragment's template path comes from props,
       // because the fragment is invisible in the DOM.
-      const { metadata, cachedTreeRoots } = walkCanvasRootFragment(
+      const { metadata, cachedPaths } = walkCanvasRootFragment(
         refOfContainer,
         0,
         props.canvasRootElementElementPath,
         props.validRootPaths,
         rootMetadataInStateRef,
-        invalidatedSceneIDsRef,
+        invalidatedPathsRef,
+        invalidatedScenesRef,
         invalidatedPathsForStylesheetCacheRef,
         props.selectedViews,
-        initComplete,
+        !initComplete,
```

Comment:
Was this wrong originally?

Reply:
Ah, you've changed what the parameter means.

Reply:
I have changed the param of the function this calls from `initComplete` (which if false would invalidate the cache for all scenes) to `invalidated` (which if true will invalidate the cache for everything from this point onwards)